src/assets/translations/szit.json

```diff
@@ -29268,7 +29268,7 @@
                   "verse": "8"
                 },
                 {
-                  "text": "Akkor, Uram, ne csak a lábamat, hanem a fejemet és a kezemet is!” – mondta Simon Péter.",
+                  "text": "„Akkor, Uram, ne csak a lábamat, hanem a fejemet és a kezemet is!” – mondta Simon Péter.",
```

Comment:
API response and text edition are aligned, but wrong.
https://szentiras.hu/api/ref/Jn13,9/SZIT

Reply:
Documented.